### DISCUSSION TITLE
Airflow event logs

### DISCUSSION THREAD

**Discussion by sou-joshi** [apache/airflow#24459] [Q&A]
We have situations where the DagRuns are deleted from the UI. Is there a way to find out who deleted? Does airflow capture these events?

I have tried using the airflow audit logs on UI, couldn't find any events for such action.

**Comment by potiuk** [comment]
Airflow never deletes them on their own. You might have some scripts running (in the official Airflow community chart there is log groomer job doing it you can look it up in configuration).
